### PR TITLE
Update readme with integrations link & a note about existing users

### PR DESCRIPTION
**Pull request by satyarohith**

### CODE REVIEW

**Review by rabbah — COMMENTED** [review]
What do you think about having two sections: 
- new to nimbella (don’t have an account yet)
- I have a nimbella account already.

**Comment by satyarohith** [comment]
> What do you think about having two sections:
> 
> * new to nimbella (don’t have an account yet)
> * I have a nimbella account already.

It will be more clear to the user so I've updated the readme.

File: README.md [hunk 1]
```diff
@@ -30,6 +36,26 @@ Use Netlify addon `nimbella` to connect your Netlify site to Nimbella.
...
+   package = "netlify-plugin-nimbella"
+   ```
+
+### Existing Nimbella User
+
+Existing users need to create a Netlify Build environment variable so the plugin can deploy the resources under their account. Follow the steps below to use your existing account with the plugin.
+
+1. **Create Netlify Build Environment Variable**
+
+   First, obtain Nimbella login token by running `nim auth export --non-expiring` if you've [`nim`](https://nimbella.io/downloads/nim/nim.html) installed or by visiting [`nimbella.com/login`](https://nimbella.com/login).
```
Comment (rabbah) [inline README.md]:
```suggestion
- Make sure you have the [Nimbella CLI `nim`](https://nimbella.io/downloads/nim/nim.html) installed. You can do that from the CLI web page or by visiting [`nimbella.com/login`](https://nimbella.com/login) to login.
- Generate a token using the CLI by running the following command:
   ```bash
   nim auth export --non-expiring
   ```
```

Comment (rabbah) [reply README.md]: the last triple ticks aren't showing up correctly in the suggestion.

File: README.md [hunk 2]
```diff
@@ -30,6 +36,26 @@ Use Netlify addon `nimbella` to connect your Netlify site to Nimbella.
...
+
+   Then, go to [`https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment`](https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment) and create an environment variable named `NIMBELLA_LOGIN_TOKEN` with your the token you just obtained.
```
Comment (rabbah) [inline README.md]:
```suggestion
   Next, visit [`https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment`](https://app.netlify.com/sites/<your-site-name>/settings/deploys#environment) to create an environment variable named `NIMBELLA_LOGIN_TOKEN` with the token you just obtained.
```

**Review by rabbah — APPROVED** [review]

**Review by rabbah — APPROVED** [review]

**Pull request merged**